snapshot/policy/policy_manager.go

```diff
@@ -98,29 +98,32 @@ func GetDefinedPolicy(ctx context.Context, rep *repo.Repository, si snapshot.Sou
 		return nil, ErrPolicyNotFound
 	}
 
-	if len(md) == 1 {
-		p := &Policy{}
+	// arbitrality pick first pick ID to return in case there's more than one
+	// this is possible when two repository clients independently create manifests at approximately the same time
+	// so it should not really matter which one we pick.
+	// see https://github.com/kopia/kopia/issues/391
```

Comment:
If they are not expected to be sorted by a particular criteria this may end up picking different policies across invocations. Which one gets chosen should be deterministic to have consistent behavior across executions / clients.

Reply:
there's really no guarantee that repo that's open at this time but not flushed at this time hasn't added yet another conflicting manifest, so this won't guarantee much, I think.